completions/apt-get

```diff
@@ -34,11 +34,13 @@ _apt_get()
                     _filedir deb
                     return
                 elif [[ $cur == *=* ]]; then
-                    COMPREPLY=( $(compgen -W "$(\
-                        apt-cache --no-generate show "${cur%%=*}" 2>/dev/null |
-                        command sed -ne \
-                            's/^Version:[[:space:]]*\([^[:space:]]\)/\1/p')" \
-                        -- "${cur#*=}") )
+                    IFS='=' read -r package cur <<< "$cur"
+                    mapfile -t COMPREPLY < <(compgen -W "$( \
```

Comment:
Process substitution will break under posix mode, and turning it off reportedly breaks other things, see 153d6d4fab1faafbd33fccb0ba8a99964f05a95b. So this should be implemented some other way.